h2o-algos/src/test/java/hex/deeplearning/DeepLearningReproducibilityTest.java

```diff
@@ -19,7 +20,7 @@
 
   @Test
   public void run() {
-    NFSFileVec ff = NFSFileVec.make(find_test_file("smalldata/junit/weather.csv"));
+    NFSFileVec ff = TestUtil.makeNfsFileVec("smalldata/junit/weather.csv");
```

Comment:
I'll create another PR, since this one is a blend of three others (which were not in master when this one was being created).